tests/unit/components/r-tab-list-item/component-test.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { describe, it, beforeEach } from 'mocha';
+import { setupComponentTest } from 'ember-mocha';
+import Ember from 'ember';
+
+const { get } = Ember;
+
+describe('Unit | Component | RUI / r-tab-list-item', function() {
+  setupComponentTest('rui/r-tab-list-item', {
+    needs: [
+      'validator:presence',
+      'validator:format'
```

Comment:
@SebastianPlace These cause the tests to fail. I'm not sure where they are coming from, what dependency we need to include here.